util/cache/cache.go

```diff
@@ -33,23 +34,32 @@ func AddCacheFlagsToCmd(cmd *cobra.Command, opts ...func(client *redis.Client))
 	sentinelAddresses := make([]string, 0)
 	sentinelMaster := ""
 	redisDB := 0
+	redisUseTLS := false
+	var redisTLSConfig *tls.Config
 	var defaultCacheExpiration time.Duration
 
 	cmd.Flags().StringVar(&redisAddress, "redis", "", "Redis server hostname and port (e.g. argocd-redis:6379). ")
 	cmd.Flags().IntVar(&redisDB, "redisdb", 0, "Redis database.")
+	cmd.Flags().BoolVar(&redisUseTLS, "redis-use-tls", false, "Whether to use TLS encryption to communicate with redis.")
```

Comment:
Is there a better name that we've used somewhere else?

Reply:
I found the following TLS specific flags:

```
--client-certificate string                     Path to a client certificate file for TLS
--client-key string                             Path to a client key file for TLS
--insecure                                      Run server without TLS
--insecure-skip-tls-verify                      If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
```

All of them however are addressed to ArgoCD itself and not one of its dependencies. As the other cache/redis related flags are `redis` and `redisdb`, I went with `redis-use-tls`.

If we want to use similar flags maybe something like `redis-insecure` and `redis-insecure-skip-tls-verify` would be a better match?

What are your thoughts?